nondex-maven-plugin/src/it/inexistent-variable-argline-it/verify.groovy

```diff
@@ -0,0 +1,3 @@
+File nondexDirectory = new File( basedir, ".nondex" );
+
+assert ! nondexDirectory.exists();
```

Comment:
No. The inexistent variables are defined in the pom.xml on line 29: https://github.com/TestingResearchIllinois/NonDex/pull/116/files/fe5de11b6de60e6775caa2decc48791b73db9e25#diff-963ead46e2ddc1dd168aeed8eeb712d6R29 and the test that checks that the properties that exist are preserved are at: https://github.com/TestingResearchIllinois/NonDex/pull/116/files/fe5de11b6de60e6775caa2decc48791b73db9e25#diff-01bdd278b3993a77e1b8f4e0f3ad386aR13